MozillaFoundation/foundation.mozilla.org
# Add a CTA link text to the homepage Take Action cards

# Description

This PR updates the homepage `Take action` card, by adding a `CTA Link Text` field, which is displayed on the lower part of the card component.

Current state:
<img width="1160" alt="image" src="https://github.com/user-attachments/assets/368fd9a1-b891-4fa5-b640-b4f2c00cf849">
<img width="988" alt="image" src="https://github.com/user-attachments/assets/ae2ec69b-1cd9-4164-875a-7b1da9c05cca">

Updated state:
![image](https://github.com/user-attachments/assets/2dc0ea92-bf2c-401a-a41d-8af87fdd96e9)
![image](https://github.com/user-attachments/assets/93ef43c1-67cf-4d32-9155-86854e1ac8b8)

# To test

1. Log in to the [preview app CMS](https://foundation-s-tp1-974-ad-bnfmtn.herokuapp.com/cms/login/?next=/cms/) with `admin2:admin2` credentials.
2. Browse the [homepage edit page](https://foundation-s-tp1-974-ad-bnfmtn.herokuapp.com/cms/pages/46/edit/). `CTA Link Text` be the last field input inside the `Take Action` card. Publish.
3. [Homepage](https://foundation-s-tp1-974-ad-bnfmtn.herokuapp.com/en/) should be updated with the latest values.

Link to sample test page: https://foundation-s-tp1-974-ad-bnfmtn.herokuapp.com/en/
Related PRs/issues: #12601
